client/src/app/collections/status-collection.coffee

```diff
@@ -0,0 +1,9 @@
+$ = require('jquery')
+Backbone = require('backbone')
+StatusModel = require('../models/status')
+
+module.exports = Backbone.Collection.extend(
+  model: StatusModel
+  comparator: (model) ->
+    -model.get('id')
```

Comment:
Just curious, now that the `Id` on a Site is back to the md5 hash, is this what we still want to use for sorting?  Do we even care about sorting on the front end?  If so, using the md5 key probably doesnt make much sense anymore since it will be arbitrary